src/components/contentful/contentfulOpenLetterForm/readLetterForm.js

```diff
@@ -59,7 +63,10 @@ const LetterContent = ({ openLetterContent, language }) => {
           {new Date(openLetterContent.created).toLocaleString()}
         </p>
         <h2>{openLetterContent.title}</h2>
-        {openLetterContent.content}
+        <MDEditor.Markdown
+          source={openLetterContent.content}
```

Comment:
Do we need to render the user's own letter as markdown at all, they probably didn't use any links or other formatting?

Reply:
You are correct, based on her reply:
> "I am not sure what do you mean. I mean if we now add a link to our letter reply, it shows as a text not a link"

We can assume that this is only for the letter reply.